job-runner/chain/chain.go

```diff
@@ -145,6 +145,8 @@ LOOP:
 			if c.CanRetrySequence(job) {
 				return false, false
 			}
+		case proto.STATE_STOPPED:
+			// Like STATE_FAIL, but ignore sequence retries.
```

Comment:
We can `return true, false` here, right? If any job was stopped, then we know the chain is done (`true`) but impossible for it to be completed (`false`) because of the stopped job.

Reply:
Yeah, `IsDone` could use some refining. I had some problems changing it too much because the traverser currently uses it to break out of the job reaper loop (<- might have to change how that works).

Question: If there's both a stopped job and a running job, is the chain "done"?